plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/client/PinotGrpcServerQueryClientTlsConfig.java

```diff
@@ -110,4 +126,15 @@ public PinotGrpcServerQueryClientTlsConfig setSslProvider(String sslProvider)
         this.sslProvider = sslProvider;
         return this;
     }
+
+    @PostConstruct
+    public void validate()
+    {
+        if (getKeystorePath().isPresent() && getKeystorePassword().isEmpty()) {
+            throw new ConfigurationException(ImmutableList.of(new Message("pinot.grpc.tls.keystore-password must set when pinot.grpc.tls.keystore-path is given")));
+        }
+        if (getTruststorePath().isPresent() && getTruststorePassword().isEmpty()) {
+            throw new ConfigurationException(ImmutableList.of(new Message("pinot.grpc.tls.truststore-password must set when pinot.grpc.tls.truststore-path is given")));
+        }
+    }
```

Comment:
sorry my earlier comment was unclear. I had meant that if `truststoreType` was non-null then `truststorePath` must also be provided.

But I notice now that `truststoreType` will always be non-null so we don't need any validation at all.

Reply:
Actually, this change unrelates to your past comment. The purpose of this validation is to prevent setting only password properties without path.

Reply:
But the code doesn't match it. It's checking if path is present and password is empty then throw. i.e. it disallows having paths without passwords. Did you mean to write `path.isEmpty() && password.isPresent()`?

Reply:
Sorry, it was opposite. It's "prevent setting only path properties without password" likes Kafka.